Library/Homebrew/cmd/test-bot.rb

```diff
@@ -11,6 +11,7 @@
 # --junit:         Generate a JUnit XML test results file.
 # --no-bottle:     Run brew install without --build-bottle
 # --keep-old:      Run brew bottle --keep-old to build new bottles for a single platform.
+# --legacy         Bulid formula from legacy Homebrew/homebrew repo.
```

Comment:
Maybe also add the same comment you made [in `pull.rb`](https://github.com/Homebrew/homebrew/blob/71cba6064f73c3e9ea46d10879f1b503bea86586/Library/Homebrew/cmd/pull.rb#L22):

```
(TODO remove it when it's not longer necessary)
```